src/Avalonia.Themes.Fluent/Controls/ItemsControl.xaml

```diff
@@ -10,8 +10,6 @@
                 CornerRadius="{TemplateBinding CornerRadius}"
                 Padding="{TemplateBinding Padding}">
           <ItemsPresenter Name="PART_ItemsPresenter"
-                          AreVerticalSnapPointsRegular="{TemplateBinding AreVerticalSnapPointsRegular}"
```

Comment:
This is a breaking change to the API of `ItemsControl` it seems. Are we OK with doing that during the RC phase?

Reply:
When do you plan to release the fix ?
In RC2 or in a nightly build before RC2 ?

Reply:
nightly builds will always be there after merging (may take some hours before available), so yes after merging you can try nightly.

Reply:
Thanks a lot @timunie and @maxkatz6